[2023-12-13T23:45:01.161+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_comments_score.get_data manual__2023-12-13T18:44:52.619180+00:00 [queued]>
[2023-12-13T23:45:01.176+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_comments_score.get_data manual__2023-12-13T18:44:52.619180+00:00 [queued]>
[2023-12-13T23:45:01.176+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-13T23:45:01.236+0500] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): get_data> on 2023-12-13 18:44:52.619180+00:00
[2023-12-13T23:45:01.251+0500] {standard_task_runner.py:57} INFO - Started process 81662 to run task
[2023-12-13T23:45:01.258+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'youtube_comments_score', 'get_data', 'manual__2023-12-13T18:44:52.619180+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/youtube_comments_score.py', '--cfg-path', '/tmp/tmpw_va75g_']
[2023-12-13T23:45:01.265+0500] {standard_task_runner.py:85} INFO - Job 10: Subtask get_data
[2023-12-13T23:45:01.690+0500] {task_command.py:416} INFO - Running <TaskInstance: youtube_comments_score.get_data manual__2023-12-13T18:44:52.619180+00:00 [running]> on host ml-vb
[2023-12-13T23:45:02.243+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='youtube_comments_score' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-13T18:44:52.619180+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-13T18:44:52.619180+00:00'
[2023-12-13T23:45:02.246+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-13T23:45:02.247+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/ml/project/flow/scripts/get_data.py']
[2023-12-13T23:45:02.281+0500] {subprocess.py:86} INFO - Output:
[2023-12-13T23:45:05.682+0500] {subprocess.py:93} INFO - 2023/12/13 23:45:05 INFO mlflow.tracking.fluent: Experiment with name 'get_data' does not exist. Creating a new experiment.
[2023-12-13T23:46:27.831+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-13T23:46:28.106+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=youtube_comments_score, task_id=get_data, execution_date=20231213T184452, start_date=20231213T184501, end_date=20231213T184628
[2023-12-13T23:46:28.160+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-13T23:46:28.266+0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
